tests/tap_tester/test_all_fields.py

```diff
@@ -0,0 +1,98 @@
+import tap_tester.connections as connections
+import tap_tester.runner as runner
+import tap_tester.menagerie as menagerie
+from base import TestPendoBase
+
+class PendoAllFieldsTest(TestPendoBase):
+    def name(self):
+        return "pendo_all_fields_test"
+    
+    def test_run(self):
+        """
+        • Verify no unexpected streams were replicated
+        • Verify that more than just the automatic fields are replicated for each stream. 
+        • verify all fields for each stream are replicated
+        • verify that the automatic fields are sent to the target
```

Comment:
This one we are testing as part of aumatic_field so not required here.

Reply:
yes, removed it.